src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1268,6 +1283,24 @@ internal string EscapePath(string path, bool ignoreSpaces = false)
             return path;
         }
 
+        internal bool UseUnixPathSeparators
+        {
+            get
+            {
+                if (PlatformUtilities.IsWindows())
```

Comment:
Personally I like it better this way (I hate super complicated if statements)...

---
In reply to: [122555606](https://github.com/Microsoft/MIEngine/pull/631#discussion_r122555606) [](ancestors = 122555606)